api/types/database.go

```diff
@@ -357,15 +368,24 @@ func (d *DatabaseV3) CheckAndSetDefaults() error {
 	// cluster ID can be extracted from the endpoint if not provided.
 	switch {
 	case strings.Contains(d.Spec.URI, rdsEndpointSuffix):
-		instanceID, region, err := parseRDSEndpoint(d.Spec.URI)
+		rdsDetails, err := parseRDSEndpoint(d.Spec.URI)
```

Comment:
That's a good idea! And that's actually why I am making this change because proxy endpoint fails parsing and prevent service to start for current version, so i need to make some adjustment on parsing.